Rule: any-kind-of-sankhara-at-all-they-see — 1 change(s), 1 file(s)

--- sujato/sutta/an/an3/an3.133_translation-en-sujato.json
+++ sujato/sutta/an/an3/an3.133_translation-en-sujato.json
@@ -1,2 +1,2 @@ an3.133:2.5
 PLI: Ye keci saṅkhārā atītānāgatapaccuppannā ajjhattā vā bahiddhā vā oḷārikā vā sukhumā vā hīnā vā paṇītā vā ye dūre santike vā, sabbe saṅkhāre: ‘netaṁ mama, nesohamasmi, na meso attā’ti evametaṁ yathābhūtaṁ sammappaññāya passati. 
-They truly see any kind of saṅkhāras at all—past, future, or present; internal or external; solid or subtle; inferior or superior; far or near: <em>all</em> saṅkhāras—with right understanding: ‘This is not mine, I am not this, this is not my self.’ 
+They truly see any kind of saṅkhāra at all—past, future, or present; internal or external; solid or subtle; inferior or superior; far or near: <em>all</em> saṅkhāras—with right understanding: ‘This is not mine, I am not this, this is not my self.’ 
